doc/ref_arch/kubernetes/README.md

```diff
@@ -5,18 +5,19 @@
 This is Kubernetes based Reference Architecture (RA-2)
 
 ## Release Information
-**Bundle: _3_**
+**Bundle: _4_**
 
 **Version: _0_**
 
-**Release Date: _15th May 2020_**
+**Release Date: _25th September 2020_**
 
 ## Bundle/Version History
 
 | Bundle.Version | Date | Note
 | --- | --- | --- |
 | 1.0-alpha | 10th January 2020 | Snezka Release|
 | 3.0 | 15th May 2020 | Baldy Release|
+| 4.0 | 25th Sep 2020 | Baraque Release|
```

Comment:
I would prefer us to format the tabs when easily doable

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1739/commits/02d57baad26960976fd5e17ff9949b5a8d119cd7